spec/models/movement_logic_spec.rb

```diff
@@ -99,4 +99,93 @@
       end
     end
   end
+
+  describe 'Pawn' do
+    it 'white pawns should be able to move up but not down' do
+      game = FactoryBot.create(:game)
+      piece = Piece.create(x: 5, y: 1, type: 'Pawn', game: game, user: game.white_player)
+      new_location = {
+        x_des: piece.x,
+        y_des: piece.y + 1
+      }
+
+      new_location2 = {
+        x_des: piece.x,
+        y_des: piece.y - 1
+      }
+      expect(piece.valid_move?(new_location)).to be true
+      expect(piece.valid_move?(new_location2)).to be false
+    end
+
+    it 'black pawns should be able to move down but not up' do
+      user = FactoryBot.create(:user)
+      game = FactoryBot.create(:game)
+      game.join_game(user, 'black')
+      game.save
+
+      piece = Piece.create(x: 5, y: 1, type: 'Pawn', game: game, user: game.black_player)
```

Comment:
I believe the y location here should also be 2 instead of 1

Reply:
Thanks for both of these! I am changing line 126 to  be x: 5 and y: 7 since that test should be for black pieces!